pkg/config/config.go

```diff
@@ -806,14 +806,15 @@ func sanitizeProjectId(src string) string {
 
 func loadDefaultEnv() error {
 	env := viper.GetString("ENV")
+
 	if env == "" {
 		env = "development"
 	}
 	filenames := []string{".env." + env + ".local"}
 	if env != "test" {
 		filenames = append(filenames, ".env.local")
 	}
-	filenames = append(filenames, ".env."+env, ".env")
+	filenames = append(filenames, ".env."+env, ".env", "supabase/.env."+env, "supabase/.env", ".env.keys", "supabase/.env.keys")
```

Comment:
Shall we avoid loading `.env.keys` in local? I think it's fine to use plaintext values in `.env.local` since it's gitignored.

If they really want to use encrypted values locally, there's an alternative to use

```
npx @dotenvx/dotenvx run -- supabase config push
```

Reply:
Yeah, there is always workaround, the friction I had in my case here: https://github.com/supabase/supabase/pull/31213

Was that since I declared the `.env` in `supabase/.env`  and that there was different environment key, but that they're depending over `SUPABASE_ENV` where `dotenvx` make it depend to `ENVIRONMENT` by default. It was either, telling me that the key was wrong to decrypt the secrets, or if I was located in the parent folder, that the keys weren't present (because they were inside `supabase/xxx` instead of being in the project root).

My guess was that if we find a `.env.keys` it probably means that the users want them loaded, and if the user have multiples of them, we want to try them until we find the one able to decrypt the secrets. But can revert tough.